### ISSUE TITLE
NTR: Spinal deficit level (sensory or motor) (Peripheral nerve, sensory, pain & headaches)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9455]
New term request Spinal deficit level (sensory or motor) (samant_110204012638): Transverse myelitis; cord level

**Comment by pnrobinson** [comment]
@MickeySegal -- is this what is meant here? 

Label: Deficit at segmental level
Definition: Refers to distribution of neurological deficits that corresponds to a deficit of a specific spinal segment and all segments below it.

This could go under [Spatial pattern HP:0012836](https://hpo.jax.org/app/browse/term/HP:0012836)

**Comment by MickeySegal** [comment]
Good.

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034556
